obvs/lenses.py

```diff
@@ -418,3 +450,34 @@ def run(self, substring: str, layers: list[int]):
         self.data["logits"] = self.data["logits"].detach()
         self.data["substring_tokens"] = substring_tokens
         self.data["layers"] = layers
+
+    def compute_surprisal_at_position(self):
```

Comment:
Can we dedupe the code by implementing this method in `BaseLogitLens`?

Reply:
Ah of course, thanks! Done